protocol/optyfi-protocol/vaults.md

```diff
@@ -1,12 +1,12 @@
 # Vaults
 
-Vaults are OptyFi smart contracts into which users deposit assets so that these assets can be deployed into strategies. Vaults enable users to share gas costs and thereby increase user profitability while also enabling more frequent strategy switching. 
+Vaults are OptyFi smart contracts into which users deposit assets in order to get these assets deployed into strategies. Vaults enable users to share gas costs and thereby increase user profitability while also enabling more frequent strategy switching. 
```

Comment:
Recommendation:
...into which users deposit  assets, in order to deploy these assets into strategies.